Automattic/wp-calypso
# Site Migration: Update Plans page for migrations flow

## Proposed Changes

* Add monthly and biennial plans to the importer upgrade step, shown only when the `migration-flow/experiment` feature flag is `true`.
* Since the upgrade step is mostly hard-coded to the Business plan, this adds panels on either side of the main plan with monthly and biennial plan details.
* Updates the loader to better resemble the new content being loaded.
* Lots of visual tweaks to better match the mockup at ay0fZYgdXxPDbZbgPlz3hu-fi-1_3529
* **Still to do**: Consider this a first draft, please. :) Updating tests, making any visual tweaks, etc. still need to be done.

## Why are these changes being made?


* p9Jlb4-fiH-p2 

## Testing Instructions



* Switch to this branch locally or via calypso.live
* Set the feature flag in session storage via the console and reload: `window.sessionStorage.setItem('flags', 'migration-flow/experiment');`
* Go through the migration flow until you get to the upgrade step
* It should look like this with the flag enabled:

![calypso localhost_3000_setup_site-migration_site-migration-upgrade-plan_siteId=239588599 siteSlug=testing2026 wordpress com from=https%3A%2F%2Fthemeflailing mystagingwebsite com%2F destination=upgrade how=myself (1)](https://github.com/user-attachments/assets/7967f286-41ff-4053-b58e-7a2b156e4d8c)

* Ensure the Monthly/Yearly/Biennially buttons add the correct products to the cart, respectively, and the prices match.
* Disable the flag in session storage: `window.sessionStorage.setItem('flags', false);` and reload the screen.
* You should see the original screen, and it should work as expected:

<img width="1509" alt="Screenshot 2024-12-04 at 3 09 18 PM" src="https://github.com/user-attachments/assets/f484593c-e323-4984-8741-e148a4198a58">

## Pre-merge Checklist
